frontend/components/contactForm.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import emailjs from "@emailjs/browser";
+import styles from "../styles/contactUs.module.css";
+import ReCAPTCHA from "react-google-recaptcha";
+
+function ContactForm() {
+  const [email, setEmail] = React.useState("");
+
+  const sendEmail = (e) => {
+    e.preventDefault();
+
+    // validating that the email is in the correct form
+    const validate =
+      /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    
+    if (validate.test(email)) {
+      emailjs
+        .sendForm(
+          process.env.REACT_APP_SERVICE_ID,
+          process.env.REACT_APP_TEMPLATE,
+          e.target,
+          process.env.REACT_APP_EMAILJS_ID
+        )
+        .then(
+          (result) => {
+            alert(
+              "Your inquiry was sent successfully. You can expect to hear a response from us within a week."
+            );
+          },
+          (error) => {
+            alert("Your inquiry failed to send. Please try again");
+          }
+        );
+      setEmail("");
+      e.target.reset();
+    } else {
+      alert("Please input a valid email");
+    }
+  };
+
+  return (
+    <form
+      method="post"
+      encType="text/plain"
+      className={styles.formStyle}
+      onSubmit={sendEmail}
+    >
+      <input
+        className={styles.formInput}
+        type="text"
+        name="name"
+        placeholder="Full Name*"
+        required
+      />
+      <br />
+      <br />
+      <input
```

Comment:
Input lines should be a bit more bolder 

Local
![image](https://user-images.githubusercontent.com/80564580/176993177-a29081ca-7099-4fde-b045-fee189e4788a.png)

Website
![image](https://user-images.githubusercontent.com/80564580/176993180-6efbc5b8-633e-4736-a922-27ebffcfbc32.png)